### DISCUSSION TITLE
Update from OMR 0.59 to 0.60

### DISCUSSION THREAD

**Discussion by Melania95** [Ysurac/openmptcprouter#3278] [General]
Hi to everyone, actually i'm using OpenMPTCProuter version 0.59.1 with kernel 5.4 and now i wish to update it on the version 0.60 with kernel 5.4 (legacy) and my question is: do I also have to update the VPS (i'm using Ubuntu Server 20.04) before upgrading to version 0.60 legacy with kernel 5.4 or does this operation have to be done only when switching to kernel 6.1? In the installation instructions I see that only on Debian VPS there is the string (wget -O - https://www.openmptcprouter.com/server/debian-x86_64.sh | KERNEL="5.4" sh) while in what concerns Ubuntu server is not there... is that the case or should the same string be installed for Ubuntu too? Finally, as I would like to try the 6.1 kernel later too, i read that not all hardware is compatible with the 6.1 kernel (or am I wrong?) I use a firewall mini/PC with an Intel N100 CPU, could I have problems doing this update? What is the difference between the two kernels essentially?
Thank you all, greetings. 
Martina.

**Comment by Fact0ryy** [comment]
Hi,

Based on my recent experience, I attempted to update to the RC version a few weeks ago. Unfortunately, the update failed for me. I believe it's due to the significant changes in the new version. After rolling back, I've now installed the latest 0.60 as a clean setup. I changed the ip of the old router and deactivated DHCP. Upon starting the new router, I gradually transferred the settings from the old router to the new one via the GUI. I noticed significant differences between them, requiring me to research how to handle them. In this environment, I have two port forwards where I access access to a few http streams and connect to PIKVM.

To preempt potential issues, I want to mention that I was previously using Ubuntu. However, due to the limited documentation and to mitigate problems, I switched to Debian on the VPS.

After encountering some issues, I now have a relatively functional environment.
